contracts/staking/PoSValidatorManager.sol

```diff
@@ -61,25 +85,41 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
         onlyInitializing
     {
         __ValidatorManager_init(settings.baseSettings);
-        __POS_Validator_Manager_init_unchained(
-            settings.minimumStakeAmount,
-            settings.maximumStakeAmount,
-            settings.minimumStakeDuration,
-            settings.rewardCalculator
-        );
+        __POS_Validator_Manager_init_unchained({
+            minimumStakeAmount: settings.minimumStakeAmount,
+            maximumStakeAmount: settings.maximumStakeAmount,
+            minimumStakeDuration: settings.minimumStakeDuration,
+            minimumDelegationFeeBips: settings.minimumDelegationFeeBips,
+            maximumStakeMultiplier: settings.maximumStakeMultiplier,
+            rewardCalculator: settings.rewardCalculator
+        });
     }
 
     // solhint-disable-next-line func-name-mixedcase
     function __POS_Validator_Manager_init_unchained(
         uint256 minimumStakeAmount,
         uint256 maximumStakeAmount,
         uint64 minimumStakeDuration,
+        uint256 minimumDelegationFeeBips,
+        uint8 maximumStakeMultiplier,
         IRewardCalculator rewardCalculator
     ) internal onlyInitializing {
         PoSValidatorManagerStorage storage $ = _getPoSValidatorManagerStorage();
+        require(minimumDelegationFeeBips > 0, "PoSValidatorManager: zero delegation fee");
```

Comment:
we should also require that `minimumDelegationFeeBips < MAXIMUM_DELEGATION_FEE_BIPS` otherwise you can initialize a manager that can't add any validators

Reply:
good point, added